src/postman/__tests__/operation.test.ts

```diff
@@ -343,7 +343,7 @@ describe('transformPostmanCollectionOperations()', () => {
                 type: 'string',
               },
               style: 'simple',
-              required: true,
+              required: false,
```

Comment:
Because now headers handling is fixed. See the input payload for this test here https://github.com/stoplightio/http-spec/blob/feat/smart-operations-merge/src/postman/__tests__/operation.test.ts#L325

There are two requests. One of them has header-a and has not header-b and second is the opposite. Merged request spec has the definition of two headers but none of them is required since that what we learned.

The change itself is caused by the fact that previously, in case of the same headers, we've been just picking full definition of the first one https://github.com/stoplightio/http-spec/blob/master/src/merge.ts#L5 (old code).

Reply:
Yeah. We assume that if a header is present in all requests then it is required.

Reply:
This is most reasonable approach IMO and should fit most of the cases. However, it is not the only one. Users may force us to implement a sort of strategy configuration object which will make them able to tweak certain behaviors.